docs/src/app/components/pages/components/FontIcon/README.md

```diff
@@ -0,0 +1,13 @@
+## Font Icons
+This component will render any [icon](https://www.google.com/design/spec/style/icons.html#) defined in any style sheets included in your project.
+We are using [Google's Material Design Icons](https://github.com/google/material-design-icons) for our documentation site along with some custom icons.
+You can use sites like IcoMoon for generating custom font files. To use FontIcons, add your stylesheet to your project and reference the icon's className in the "className" prop.
+
+We also support [Google's Material Icons](http://google.github.io/material-design-icons/) as seen in the third block of code. If you're using the material icons, be sure to include the link to the font icon file in your head section:
+```html
+<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+```
+
+To see available Material Icons, go to [material icons library](https://design.google.com/icons/). The names are in snake_case format, for example: find_in_page
```

Comment:
When calling google's material icons, their names are store using that convention instead of camel case. For example, instead of ActionGrade, we would use action-grade when we wanted to call that svg from google's icons. I guess it should be phrased better